Example, readme

lisapro opened the pull request.

knl88 reviewed (approved):
Looks really nice! Did you run it with the new key?  

In the notebook the code says RW, but the output shows NB. Perhaps either evaluate the notebook again, or clear the output.

lisapro commented:
> In the notebook the code says RW, but the output shows NB. Perhaps either evaluate the notebook again, or clear the output.

thanks for pointing out. I did not test it with the new key (I actually don't have it). Could you test? I removed the output anyways, so it is not  confusing. I did test the notebook on JupyterHub with our niva key

knl88 commented:
> > In the notebook the code says RW, but the output shows NB. Perhaps either evaluate the notebook again, or clear the output.
> 
> thanks for pointing out. I did not test it with the new key (I actually don't have it). Could you test? I removed the output anyways, so it is not confusing. I did test the notebook on JupyterHub with our niva key

Works fine, but I updated the paths to reflect the once the account have access to.

lisapro commented:
> > > In the notebook the code says RW, but the output shows NB. Perhaps either evaluate the notebook again, or clear the output.
> > 
> > 
> > thanks for pointing out. I did not test it with the new key (I actually don't have it). Could you test? I removed the output anyways, so it is not confusing. I did test the notebook on JupyterHub with our niva key
> 
> Works fine, but I updated the paths to reflect the once the account have access to.

then RW/INLET/SBE38/TEMPERATURE/RAW should also be there?

knl88 commented:
> > > > In the notebook the code says RW, but the output shows NB. Perhaps either evaluate the notebook again, or clear the output.
> > > 
> > > 
> > > thanks for pointing out. I did not test it with the new key (I actually don't have it). Could you test? I removed the output anyways, so it is not confusing. I did test the notebook on JupyterHub with our niva key
> > 
> > 
> > Works fine, but I updated the paths to reflect the once the account have access to.
> 
> then RW/INLET/SBE38/TEMPERATURE/RAW should also be there?

Yes, but it isn't :thinking: not sure why?

27am commented:
Nice job @lisapro! I also tested with the ntnu-service-account key and works nicely. Not sure either why `RW/INLET/SBE38/TEMPERATURE/RAW` isn't there
